[2024-11-19T09:50:47.186+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-19T09:50:47.197+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: bruno_mars.save_youtube_data manual__2024-11-19T09:49:47.043856+00:00 [queued]>
[2024-11-19T09:50:47.203+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: bruno_mars.save_youtube_data manual__2024-11-19T09:49:47.043856+00:00 [queued]>
[2024-11-19T09:50:47.204+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-11-19T09:50:47.216+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): save_youtube_data> on 2024-11-19 09:49:47.043856+00:00
[2024-11-19T09:50:47.220+0000] {standard_task_runner.py:72} INFO - Started process 194 to run task
[2024-11-19T09:50:47.223+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'bruno_mars', 'save_youtube_data', 'manual__2024-11-19T09:49:47.043856+00:00', '--job-id', '85', '--raw', '--subdir', 'DAGS_FOLDER/dag2.py', '--cfg-path', '/tmp/tmpn3wefpm_']
[2024-11-19T09:50:47.225+0000] {standard_task_runner.py:105} INFO - Job 85: Subtask save_youtube_data
[2024-11-19T09:50:47.240+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-11-19T09:50:47.263+0000] {task_command.py:467} INFO - Running <TaskInstance: bruno_mars.save_youtube_data manual__2024-11-19T09:49:47.043856+00:00 [running]> on host 84d1fd291955
[2024-11-19T09:50:47.321+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='bruno_mars' AIRFLOW_CTX_TASK_ID='save_youtube_data' AIRFLOW_CTX_EXECUTION_DATE='2024-11-19T09:49:47.043856+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-19T09:49:47.043856+00:00'
[2024-11-19T09:50:47.322+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-19T09:50:47.367+0000] {logging_mixin.py:190} INFO - Successfully connected to MongoDB.
[2024-11-19T09:50:47.371+0000] {logging_mixin.py:190} INFO - Existing data found, moving to historical collection...
[2024-11-19T09:50:47.373+0000] {logging_mixin.py:190} INFO - Existing data moved to historical collection.
[2024-11-19T09:50:47.375+0000] {logging_mixin.py:190} INFO - New data for channel statistics successfully saved to MongoDB.
[2024-11-19T09:50:47.375+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-11-19T09:50:47.381+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-19T09:50:47.382+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=bruno_mars, task_id=save_youtube_data, run_id=manual__2024-11-19T09:49:47.043856+00:00, execution_date=20241119T094947, start_date=20241119T095047, end_date=20241119T095047
[2024-11-19T09:50:47.396+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-19T09:50:47.409+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-19T09:50:47.410+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
